docs/schemaform/form-config.md

```diff
@@ -66,9 +66,23 @@ Forms are created by creating a page that uses FormApp from the schemaform folde
           // Objects that start with view: will not be sent, but their children will be merged
           // into the parent object and will be sent
           // These can be used to remove fields from what is sent to the server, like if we have
-          // a question that is only used to reveal other questions
+          // a question that is only used to reveal other questions or group related questions
+          // together to be conditionally revealed that aren't in an object in the schema
```

Comment:
Perhaps this could be explained better? I'm thinking for situations like the 5490's `previousBenefits` page where some options (`chapter35`, `transferOfEntitlement`, etc.`) were revealed under `view:claimedSponsorService`, but siblings of the other options (`chapter31`, `disability`, etc.) in the schema. In that case, I was able to `expandUnder` for each of them, but if `hideIf` is required, these could be grouped in a `view:someName` and the `hideIf` applied to `view:someName` only without affecting the end result sent to the api.

Does this explanation make the above use case clear?